src/fidesops/models/privacy_request.py

```diff
@@ -50,6 +51,30 @@
 logger = logging.getLogger(__name__)
 
 
+class ManualAction(BaseSchema):
+    """Surface how to manually retrieve or mask data in a database-agnostic way
+
+    "locators" are similar to the SQL "WHERE" information.
+    "get" contains a list of fields that should be retrieved from the source
+    "update" is a dictionary of fields and the value they should be replaced with.
+    """
+
+    locators: Dict[str, Any]
+    get: Optional[List[str]]
+    update: Optional[Dict[str, Any]]
```

Comment:
The UI for resuming/restarting requests hasn't been designed yet, so I'm just trying to store this in as flexible a way as possible for now.

Reply:
hm, well I'm not creating any of those currently, but I could add a validator for developers to catch if at least one isn't being populated.